app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -201,6 +201,17 @@ export const TokenPage = (): React.ReactElement => {
             <Stack.Item growthFactor={1} shrinkFactor={1}>
               <Stack direction={Direction.Vertical} contentAlignmentResponsive={{ base: Alignment.Center, medium: Alignment.Start }}>
                 <Text variant='header2'>{collectionToken.name}</Text>
+                <Stack direction={Direction.Horizontal} contentAlignmentResponsive={{ base: Alignment.Center, medium: Alignment.Start }}>
+                  <IconButton variant={'tertiary'} icon={<Box height='1rem' width='1rem' variant='unrounded'><Image variant='unrounded' source={'/assets/icon-info.svg'} alternativeText={'info'} /></Box>} target={`https://tokenhunt.io/collections/${collectionToken.registryAddress}/tokens/${tokenId}`} />
+                  {latestTransfer
```

Comment:
why are you going to the transfer? you should just be going to the token on these sites it has nothing to do with the transfer